docs/en/offerings/spc/setup/traffic_segregation.md

```diff
@@ -9,26 +9,24 @@ lang: en
 
 The goal is to set up an AWS PrivateLink to privately access your Shotgun site.
 
-## Setup a VPC within the us-east-1 AWS region if needed
-
-Using the AWS Console:
-
-* Create your VPC
+## Set up PrivateLink to Shotgun
 
-![Create VPC](../images/spc-vpc-create.png)
+* Ask Shotgun support to provide you with the Shotgun PrivateLink service name for your AWS region.
 
-* Create your private subnets
+* Add a new VPC Endpoint in your VPC
 
-![Create subnets](../images/spc-subnet-create.png)
+* For the security group, Shotgun service only requires the inbound port tcp/443 to be open.
 
-## Set up PrivateLink to Shotgun
+* Please contact Shotgun Support to get the endpoint approved, so that you can start using it
 
-* Add a new VPC Endpoint in your us-east-1 VPC
+![Create endpoint](../images/spc-endpoint-create_privatelink.png)
 
-* The service name is `com.amazonaws.vpce.us-east-1.vpce-svc-001d9eddae4b841b8`
+## Split Horizon DNS
 
-* For the security group, Shotgun only requires the inbound port tcp/443 to be open
+You need to configre your office DNS server to resolve your shotgun site to your Shotgun VPC Endpoint DNS name.
 
-* Please contact Shotgun Support to get the endpoint approved, so that you can start using it
+Example DNS entry
 
-![Create endpoint](../images/spc-endpoint-create_privatelink.png)
+```
+mystudio-staging.shotgunstudion.com   CNAME   vpce-048447456a4f57e14-1j3wh50q.vpce-svc-0b054415458f57634.us-west-2.vpce.amazonaws.com
```

Comment:
mystudio-staging.shotgunstudion.com should be mystudio-staging.shotgunstudio.com